safe-global/safe-homepage
# feat: inline play Hero videos

## What it solves

Keeps the videos from playing in a fullscreen modal disrupting the mobile experience.

## How this PR fixes it

- Added the `playsInline` attribute to the `<video>` tag.
- Removed the `.mp4` files, keeping only _webm_ with a _.hevc.mp4_ fallback.
- Moved the Core related assets inside `/Core` folder.

## How to test

Navigate from https://safe-web-landing.dev.5afe.dev/ to https://safe-web-landing.dev.5afe.dev/core in a mobile browser and observe the Intro videos playing in the layout rather than in fullscreen
